### PR TITLE
Setup mako for uploading load test metrics to server

### PR DESCRIPTION
**Pull request by chaodaiG**
This PR injects performance test metrics to https://mako.dev, so that the results can be visualized there as well. 

This had been tested locally and performance metrics was uploaded to mako server, which could be visualized at https://mako.dev/benchmark?benchmark_key=5698752440434688. (Note: for faster turnaround time, the data for this was from 1000 publishes instead of 100000 in real test)

/assign @sethvargo @mikehelmick 
/cc @stati0n

### CODE REVIEW

File: internal/performance/helper.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+	mpb "github.com/google/mako/spec/proto/mako_go_proto"
+)
+
+const (
+	benchmarkConfigFile = "export_test_benchmark.config"
+)
+
+// setup sets up client used for performance test
+func setup(tb testing.TB) (*quickstore.Quickstore, func(context.Context)) {
+	var (
```
Comment (mikehelmick) [inline internal/performance/helper.go:45]: just
```
var microservice string
```

Comment (chaodaiG) [reply internal/performance/helper.go]: done

File: internal/performance/helper.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+		microservice string
+	)
+
+	benchmarkConfig := &mpb.BenchmarkInfo{}
+
+	{
+		p := os.Getenv("MAKO_PORT")
```
Comment (mikehelmick) [inline internal/performance/helper.go:49]: env var configuration should use a config struct
https://github.com/google/exposure-notifications-server/blob/main/internal/publish/config.go#L39

and then we use a library to parse the environment

File: internal/performance/helper.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	benchmarkConfigFile = "export_test_benchmark.config"
+)
+
+// setup sets up client used for performance test
+func setup(tb testing.TB) (*quickstore.Quickstore, func(context.Context)) {
+	var (
+		microservice string
+	)
+
+	benchmarkConfig := &mpb.BenchmarkInfo{}
```
Comment (mikehelmick) [inline internal/performance/helper.go:46]: declare this closer to where it us used

File: internal/performance/helper.go (hunk 2 above)
Comment (chaodaiG) [reply internal/performance/helper.go]: sg, done

File: internal/performance/helper.go (hunk 3 above)
Comment (chaodaiG) [reply internal/performance/helper.go]: After using envconfig the closures don't seem to be necessary, removed closures and reordered a little bit

**Pull request merged**